DrawerKitDemo/DrawerKitDemo/PresenterViewController.swift

```diff
@@ -37,6 +37,7 @@ private extension PresenterViewController {
         configuration.upperMarkGap = 100 // default is 40
         configuration.lowerMarkGap =  80 // default is 40
         configuration.maximumCornerRadius = 15
+        configuration.neverDismissCollapsedDrawer = true
```

Comment:
If changing from `neverDismissCollapsedDrawer` to `canDismissCollapsedDrawer`, then the value here should be `false`.